nixos/modules/services/networking/znc.nix

```diff
@@ -350,6 +350,8 @@ in
 
   config = mkIf cfg.enable {
 
+    networking.firewall.allowedTCPPorts = [ cfg.port ];
```

Comment:
Currently we have do not open ports automatically by convention just if the server is enabled.
Users might want to use the service, but restrict the port only for specific source ips / network interfaces or don't expose it at all.
Some services have therefor an additional property called `openPorts` or `openFirewall` in the service itself to enable this line.

Reply:
That makes sense. I've added an `openFirewall` option like the one in the Redis configuration.